deploy/contents/k8s/fsx-csi/resources/fsx-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fsx-claim
```

Comment:
rename this pvc to `cp-system-pvc` to be compatible with `cp-aws-native` resources definition